Sources/ArgumentsUtils.swift

```diff
@@ -129,20 +141,24 @@ let bundledTemplatesPath = Path(ProcessInfo.processInfo.arguments[0]).parent() +
  - returns: The Path matching the template to find
  */
 func findTemplate(prefix: String, templateShortName: String, templateFullPath: String) throws -> Path {
-  guard templateFullPath.isEmpty else {
+  switch (templateFullPath, templateShortName) {
+  case ("", ""): throw TemplateError.noTemplateProvided
```

Comment:
Nitpicking on code style, but I'd rather do:

```
switch (!templateFullPath.isEmpty, !templateShortName.isEmpty) {
case (false, false)
  throw TemplateError.noTemplateProvided
…
```
That is:

 - Switch over `!.isEmpty` rather than comparing the strings with `""` and `_`, I think it's more performant and will make it (arguably) more readable?
 - Go to the next line after each `case …:` to be consistent, even for small case bodies.